qa/tasks/cephfs/fuse_mount.py

```diff
@@ -260,6 +260,8 @@ def _mountpoint_exists(self):
 
     def umount(self):
         if not self.is_mounted():
+            self.mounted = False
```

Comment:
This is useless here IMO, since the is_mounted() will check the self.mounted and it is False, or we shouldn't be here.

Reply:
True.

Reply:
Fixed.